packages/server/graphql/public/types/PokerTemplate.ts

```diff
@@ -3,11 +3,17 @@ import {PokerTemplateResolvers} from '../resolverTypes'
 const RECOMMENDED_TEMPLATES = ['estimatedEffortTemplate']
 
 const PokerTemplate: PokerTemplateResolvers = {
-  category: (_source, _args, _context) => {
-    return 'estimation'
+  __isTypeOf: ({type}) => type === 'poker',
```

Comment:
-1 Let's pull the typedefs out of `_legacy.graphql` too if we're migrating these to SDL.

Reply:
I'm all for that!